dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/PagesVersionHandler.java

```diff
@@ -57,32 +57,46 @@ private static synchronized String loadPagesVersion(){
         return PagesVersionHandler.DEFAULT_VERSION;
     }
 
-    public static boolean isPages30Loaded(){
-        if(PagesVersionHandler.LOADED_SPEC_LEVEL.equals("3.0")) {
+    public static boolean isPages30Loaded() {
+        if (PagesVersionHandler.LOADED_SPEC_LEVEL.equals("3.0")) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isPages31Loaded() {
+        if (PagesVersionHandler.LOADED_SPEC_LEVEL.equals("3.1")) {
             return true;
         }
         return false;
     }
 
-    public static boolean isPages31Loaded(){
-        if(PagesVersionHandler.LOADED_SPEC_LEVEL.equals("3.1")) {
+    public static boolean isPages31OrHigherLoaded() {
+        if (Double.parseDouble(PagesVersionHandler.LOADED_SPEC_LEVEL) >= 3.1) {
             return true;
         }
         return false;
     }
 
-    public static boolean isPages31OrHigherLoaded(){
-        if(Double.parseDouble(PagesVersionHandler.LOADED_SPEC_LEVEL) >= 3.1) {
+    public static boolean isPages30OrLowerLoaded() {
+        if (Double.parseDouble(PagesVersionHandler.LOADED_SPEC_LEVEL) <= 3.0) {
             return true;
         }
         return false;
     }
 
-    public static boolean isPages30OrLowerLoaded(){
-        if(Double.parseDouble(PagesVersionHandler.LOADED_SPEC_LEVEL) <= 3.0) {
+    public static boolean isPages31OrLowerLoaded() {
```

Comment:
We should update the copyright date range header to include 2023 and clean up the contributors.